dna-src/test/unit_test/categories_test.js

```diff
@@ -14,11 +14,11 @@ module.exports = (app) => {
 
   test('Category test', (t) => {
 
-    const app_address = app.call("provider", "main", "register_app", App_Config_1);
+    const app_address = app.call("provider", "main", "register_app_bundle", App_Config_1);
```

Comment:
Since you updated all the test and updated to the register_app. I would request you to remove the register_app_bundle().
We don't need to keep it around for just two test cases.